lib/sync/commits.js

```diff
@@ -825,8 +827,9 @@ export async function syncCommits(primaryPR, secondaryPR) {
       target: {
         baseBranch: targetPRInfo.base.ref,
         branch: targetPRInfo.head.ref,
-        commitSha: targetCommit.sha,
-        commitTreeSha: targetCommit._.commit.tree.sha,
```

Comment:
`targetCommit._` was undefined in the case of `targetCommit` being set to `targetCommits[0]._.parents[0]`. This fix changes it to just GET the tree sha on the fly using the `commitSha`